packages/frameworks/src/frameworks.ts

```diff
@@ -17,6 +17,73 @@ const { readdir, readFile, unlink } = promises;
  */
 
 export const frameworks = [
+  {
+    name: 'Remix',
+    slug: 'remix',
+    demo: 'https://remix.examples.vercel.com',
+    logo: 'https://raw.githubusercontent.com/vercel/vercel/main/packages/frameworks/logos/remix.svg',
+    tagline: 'Build Better Websites',
+    description: 'A new Remix app — the result of running `npx create-remix`.',
+    website: 'https://remix.run',
+    detectors: {
+      every: [
+        {
+          path: 'package.json',
+          matchContent:
+            '"(dev)?(d|D)ependencies":\\s*{[^}]*"remix":\\s*".+?"[^}]*}',
+        },
+      ],
+    },
+    settings: {
+      installCommand: {
+        placeholder: '`yarn install` or `npm install`',
+      },
+      buildCommand: {
+        value: 'remix build',
+        placeholder: '`npm run build` or `remix build`',
+      },
+      devCommand: {
+        value: 'remix dev',
+        placeholder: 'remix dev',
+      },
+      outputDirectory: {
+        value: 'public',
+      },
+    },
+    dependency: 'remix',
+    getFsOutputDir: async () => 'public',
+    getOutputDirName: async () => 'public',
+    defaultRoutes: [
+      {
+        src: '^/build/(.*)$',
+        headers: { 'cache-control': 'public, max-age=31536000, immutable' },
+        continue: true,
+      },
+      {
+        handle: 'filesystem',
+      },
+      {
+        src: '/(.*)',
+        dest: '/api',
+      },
+    ],
+    defaultRewrites: [
+      {
+        source: '/(.*)',
+        regex: '/(.*)',
+        destination: '/api',
+      },
+    ],
+    defaultHeaders: [
+      {
+        source: '^/build/(.*)$',
+        regex: '^/build/(.*)$',
+        headers: [
+          { key: 'cache-control', value: 'public, max-age=31536000, immutable' },
+        ],
+      },
+    ]
+  },
```

Comment:
Lets move this down so it doesn't show up as the first item in the template list.

We also have a `sort` property that should handle the sorting for front.

Reply:
> Lets move this down so it doesn't show up as the first item in the template list.

That's not how it works, though? Blitz.js is first, yet it's not first there.

Let me check the property you mentioned.

Reply:
Updated the sorting to match our [Getting Started Docs](https://vercel.com/docs/get-started)!

Reply:
Ah! Moved it down a bit for correctness!